### PR TITLE
add override to avoid warning.

### PR DESCRIPTION
**Pull request by Dragonliu2018**
### What problem does this PR solve?

Add override to avoid warning.

### Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [x] Refactoring
- [ ] Performance Improvement
- [ ] Test cases
- [ ] Python SDK impacted, Need to update PyPI
- [ ] Other (please describe):

### CODE REVIEW

File: src/storage/meta/entry/db_entry.cppm [hunk 1]
```diff
@@ -61,7 +61,7 @@ public:
 
     static UniquePtr<DBEntry> Deserialize(const nlohmann::json &db_entry_json, BufferManager *buffer_mgr);
 
-    virtual void MergeFrom(BaseEntry &other); // TODO: fix warning
+    virtual void MergeFrom(BaseEntry &other) override;
```
Comment (JinHai-CN) [inline src/storage/meta/entry/db_entry.cppm:64]: Remove virtual and change overide to final

File: src/storage/meta/entry/table_entry.cppm [hunk 2]
```diff
@@ -158,7 +158,7 @@ public:
 
     static UniquePtr<TableEntry> Deserialize(const nlohmann::json &table_entry_json, TableMeta *table_meta, BufferManager *buffer_mgr);
 
-    virtual void MergeFrom(BaseEntry &other);
+    virtual void MergeFrom(BaseEntry &other) override;
```
Comment (JinHai-CN) [inline src/storage/meta/entry/table_entry.cppm:161]: Same as above

File: src/storage/meta/entry/db_entry.cppm (hunk 1 above)
Comment (JinHai-CN) [reply src/storage/meta/entry/db_entry.cppm:64]: Also consider adding 'final' after the derived class name.

**Review by JinHai-CN — CHANGES_REQUESTED** [review]

File: src/storage/meta/entry/db_entry.cppm (hunk 1 above)
Comment (Dragonliu2018) [reply src/storage/meta/entry/db_entry.cppm]: done

File: src/storage/meta/entry/table_entry.cppm (hunk 2 above)
Comment (Dragonliu2018) [reply src/storage/meta/entry/table_entry.cppm]: done

**Review by JinHai-CN — APPROVED** [review]

**Comment by JinHai-CN** [comment]
LGTM

**Pull request merged**